s3/pkg/meta/db/drivers/tidb/object.go

```diff
@@ -1,20 +1,23 @@
 package tidbclient
 
 import (
+	"context"
 	"database/sql"
-	. "github.com/opensds/multi-cloud/s3/pkg/meta/types"
+	"encoding/hex"
+	"encoding/json"
 	"math"
 	"strconv"
-	"encoding/hex"
+	"time"
+
+	. "github.com/opensds/multi-cloud/s3/pkg/meta/types"
+	pb "github.com/opensds/multi-cloud/s3/proto"
+	log "github.com/sirupsen/logrus"
 	"github.com/xxtea/xxtea-go/xxtea"
-	. "github.com/opensds/multi-cloud/s3/error"
-	"context"
 )
 
 func (t *TidbClient) GetObject(ctx context.Context, bucketName, objectName, version string) (object *Object, err error) {
-	var ibucketname, iname, customattributes, acl, lastModifiedTime string
+	var sqltext, ibucketname, iname, customattributes, acl, lastModified string
 	var iversion uint64
-	var sqltext string
 	var row *sql.Row
 	if version == "" {
 		sqltext = "select * from objects where bucketname=? and name=? order by bucketname,name,version limit 1;"
```

Comment:
do we need to add ASC or DESC here ? i am thinking i would like to see my versioned objects in descending order of versions, since the later versions are more recent...like 
b1-o1-v3
b1-01-v2
etc.

Reply:
Same as above, see [#713](https://github.com/opensds/multi-cloud/issues/713) please.